ngx_rtmp_stat_module.c

```diff
@@ -866,10 +866,12 @@ ngx_rtmp_stat_live(ngx_http_request_t *r, ngx_chain_t ***lll,
 
                 NGX_RTMP_STAT_L("}");
             }
+
+            NGX_RTMP_STAT_L(",");
```

Comment:
If there is a stream or some streams in the chain in the bucket n array, there will be an extra comma in json.